corporate/lib/stripe.py

```diff
@@ -1951,6 +2001,11 @@ def is_valid_plan_tier_switch(self, current_plan_tier: int, new_plan_tier: int)
         # TBD
         return False
 
+    @override
+    def has_billing_access(self) -> bool:
+        # TBD
+        return False
```

Comment:
Changed these values, since we do know what is correct here:

``` diff
diff --git a/corporate/lib/stripe.py b/corporate/lib/stripe.py
index eca338967e..a2a61e7d9e 100644
--- a/corporate/lib/stripe.py
+++ b/corporate/lib/stripe.py
@@ -2003,8 +2003,9 @@ class RemoteRealmBillingSession(BillingSession):  # nocoverage
 
     @override
     def has_billing_access(self) -> bool:
-        # TBD
-        return False
+        # We don't currently have a way to authenticate a remote
+        # session that isn't authorized for billing access.
+        return True
 
 
 class RemoteServerBillingSession(BillingSession):  # nocoverage
@@ -2194,8 +2195,9 @@ class RemoteServerBillingSession(BillingSession):  # nocoverage
 
     @override
     def has_billing_access(self) -> bool:
-        # TBD
-        return False
+        # We don't currently have a way to authenticate a remote
+        # session that isn't authorized for billing access.
+        return True
 
 
 def stripe_customer_has_credit_card_as_default_payment_method(
```